README.md

```diff
@@ -7,6 +7,7 @@ This is the repository for [Kubernetes DNS](http://kubernetes.io/docs/admin/dns/
 ## Subprojects
 
 * [sidecar](docs/sidecar/README.md)
+* [dnsmasq container](images/dnsmasq/RELEASES.md)
```

Comment:
Probably create a short README.md for dnsmasq and link it here? RELEASE.md does not mention what this dnsmasq is...